thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/dashboard/resources/EmailResource.java

```diff
@@ -81,6 +69,74 @@ public Response deleteByAlertId(@PathParam("alertId") Long alertId) {
     return Response.ok().build();
   }
 
+  /**
+   * update alert config's cron and activation by id
+   * @param id alert config id
+   * @param cron cron expression for alert
+   * @param isActive activate or not
+   * @return Response
+   * @throws Exception
+   */
+  @PUT
+  @Path("/alert/{id}")
+  public Response updateAlertConfig(@NotNull @PathParam("id") Long id,
+      @QueryParam("cron") String cron, @QueryParam("isActive") Boolean isActive) throws Exception {
```

Comment:
there already is an enable/disable endpoint in AlertJobResource.

Reply:
AlertJobResource. Add this new class to EmailResources.  We can consider rename the file later. 
For now just keep it here in the repo.